packages/sdk-node/src/clerkClient.ts

```diff
@@ -1,29 +1,25 @@
-import type { ClerkOptions, VerifyTokenOptions } from '@clerk/backend';
-import { createClerkClient, verifyToken as _verifyToken } from '@clerk/backend';
+import type { ClerkOptions } from '@clerk/backend';
+import { createClerkClient, verifyToken } from '@clerk/backend';
 
 import { createClerkExpressRequireAuth } from './clerkExpressRequireAuth';
 import { createClerkExpressWithAuth } from './clerkExpressWithAuth';
 import { loadApiEnv, loadClientEnv } from './utils';
 
-type ExtendedClerk = ReturnType<typeof createClerkClient> & {
+type ClerkClient = ReturnType<typeof createClerkClient> & {
   expressWithAuth: ReturnType<typeof createClerkExpressWithAuth>;
   expressRequireAuth: ReturnType<typeof createClerkExpressRequireAuth>;
-  verifyToken: (token: string, verifyOpts?: Parameters<typeof _verifyToken>[1]) => ReturnType<typeof _verifyToken>;
+  verifyToken: typeof verifyToken;
 };
 
 /**
  * This needs to be a *named* function in order to support the older
  * new Clerk() syntax for v4 compatibility.
  * Arrow functions can never be called with the new keyword because they do not have the [[Construct]] method
  */
-export function Clerk(options: ClerkOptions): ExtendedClerk {
+export function Clerk(options: ClerkOptions): ClerkClient {
   const clerkClient = createClerkClient(options);
   const expressWithAuth = createClerkExpressWithAuth({ ...options, clerkClient });
   const expressRequireAuth = createClerkExpressRequireAuth({ ...options, clerkClient });
-  const verifyToken = (token: string, verifyOpts?: VerifyTokenOptions) => {
-    const issuer = (iss: string) => iss.startsWith('https://clerk.') || iss.includes('.clerk.accounts');
-    return _verifyToken(token, { issuer, ...options, ...verifyOpts });
-  };
```

Comment:
Before this change, we supported using the following:
```
import { clerkClient } from '@clerk/clerk-sdk-node';
clerkClient.verifyToken('token');
```

With the above change the customers using the `@clerk/clerk-sdk-node` and `clerkClient.verifyToken` 
would have to pass an issuer and some options for the function to work.
Customers were using the previous one since we had a reported issue to fix this not a long time ago: https://github.com/clerk/javascript/issues/2280

Was this change intentional or accidental?

cc: @nikosdouvlis , @SokratisVidros

Reply:
@dimkl the issuer check has been removed entirely

Reply:
ooh. I see. thanks for the clarification 😄 
What was the reasoning behind removing the `issuer` check? Do we think that the rest of the claims are sufficient?